include/swift/Option/Options.td

```diff
@@ -549,6 +549,11 @@ def disable_bridging_pch : Flag<["-"], "disable-bridging-pch">,
 def lto : Joined<["-"], "lto=">,
   Flags<[FrontendOption, NoInteractiveOption]>,
   HelpText<"Specify the LTO type to either 'llvm-thin' or 'llvm-full'">;
+  
+def lto_library : Separate<["-"], "lto_library">,
```

Comment:
Tiny nitpick: we usually prefer dashes to separate pieces of arguments, e.g. `-lto-library`.

Reply:
Ah, I see this is passed down to the linker. Still, I think we should promote linker-style arguments to Swift-style when we expose them from the Driver.

Reply:
Simple enough. Done 👍